pytest_fixtures/component/provision_vmware.py

```diff
@@ -107,3 +109,19 @@ def module_vmware_image(
         uuid=settings.vmware.image_name,
         password=settings.provisioning.host_root_password,
     ).create()
+
+
+@pytest.fixture
+def provisioning_vmware_host(pxe_loader, vmwareclient):
+    """Fixture to check out blank VM on VMware"""
+    vm_boot_firmware = 'efi' if pxe_loader.vm_firmware == 'uefi' else 'bios'
+    host_data = Broker(
+        workflow='deploy-blank-vm-vcenter',
+        artifacts='last',
+        vm_network=settings.provisioning.vlan_id,
+        vm_boot_firmware=vm_boot_firmware,
+    ).execute()
+    yield host_data
+    # delete the host
+    vmware_host = VMWareVirtualMachine(vmwareclient, name=f'{host_data["name"]}')
+    vmware_host.delete()
```

Comment:
how about confirming after delete if host is really deleted on vcenter with wrapanapi? like 
`assert not VMWareVirtualMachine(vmwareclient, name=host_data['name'])`